guide/daily-spending-wallet/privacy.md

```diff
@@ -244,23 +157,134 @@ An advantage here is the risk of an unauthorized person revealing their informat
+## Data transfer
 
-<!--
-There is no perfect solution to guarantee 100% privacy that lasts forever because things can be revealed over time. Since transactions are forever public, even if all precautions are taken at the time of payment to ensure the highest degree of anonymity, future behaviors of the wallet owner or transacting parties can still degrade previously attained privacy. A high amount of diligence is necessary whenever users are transacting with bitcoin. The product should be able to guide, inform, and prevent them against privacy degrading actions.
--->
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/privacy/network-settings.png"
+   retina = "/assets/images/guide/daily-spending-wallet/privacy/network-settings@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/privacy/network-settings-big.png"
+   alt-text = "Application network settings"
+   caption = "Option to enable Tor in settings."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Internet service providers and similar services that handle data in transit may be able to gather or infer information about the user from the traffic. Tools like [VPNs](https://en.wikipedia.org/wiki/Virtual_private_network) and [Tor](https://en.wikipedia.org/wiki/Tor_(network)) can prevent this by adding additional layers of encryption and masking user activity. Keep in mind that using Tor provides better privacy, but can noticeably slow down the experience (especially initial startup).
+
+</div>
+
+## Lightning network
+
+On the lightning network, a payment is only stored by the respective sender and receiver, and only as long as the channel in which the payment was made is open. However, opening and closing channels requires entries on the bitcoin blockchain, and those are also publicly stored forever. Additionally, lightning nodes are always online and usually directly tied to a single wallet, providing another data point. For a detailed analysis of privacy on lightning, see the [Security and privacy chapter](https://github.com/lnbook/lnbook/blob/develop/16_security_privacy_ln.asciidoc) in the Mastering the Lightning Network book.
+
+#### Be intentional when sharing static lightning identifiers
+
+Lightning node Ids, lightning addresses, and LNURL-Pay invoices (see [Payment request formats](https://bitcoin.design/guide/how-it-works/payment-request-formats/)) are examples of endpoints that can be used to generate many lightning invoices. While this is convenient for users, it is bad for privacy. For example, placing a lightning address on a website or social media profile makes it trivial to create a direct connection between the lightning node and the owner of the website or profile.
+
```

Comment:
```suggestion

#### Be conscious about data leaks when generating invoices

When paying an invoice the origin is difficult / not possible to determine by the receiver. Generating and sharing invoices though will have encoded information about your node. In a daily spending wallet, it may not be practical to avoid this type of transaction but it may be something you may want to inform the wallet owner of during onboarding.
```

Reply:
Can you explain this a bit? I don't quite get it.